tests/testsuite/cargo_remove/update_lock_file/mod.rs

```diff
@@ -33,5 +35,7 @@ fn case() {
         .stdout_matches(str![""])
         .stderr_matches(file!["stderr.term.svg"]);
 
-    assert_ui().subset_matches(current_dir!().join("out"), &project_root);
+    let expected = Project::from_expected(current_dir!().join("out"));
```

Comment:
The dynamic check seems fine to me, though I am a bit unsure since most UI tests in Cargo only asserts one command invocation.

@epage, do you have any input for this?

Reply:
Note that it's not *just* about expected output in the tests. In `cargo_add::locked_unchanged`, cargo itself complains that the input lockfile looks corrupt with its mismatched checksum.